R/badge.R

```diff
@@ -144,6 +157,35 @@ use_binder_badge <- function(ref = git_default_branch(), urlpath = NULL) {
 
   invisible(TRUE)
 }
+#' @rdname badges
+#' @export
+use_r_universe_badge <- function(repo_spec = NULL) {
+  check_is_package("use_r_universe_badge()")
+  pkg <- project_name()
+
+  if (is.null(repo_spec)) {
+    this_env <- current_call()
+    tryCatch(
+      github_url <- github_url(),
+      error = function(e) {
+        ui_abort(
+          c(
+            "x" = "Can't determine the R-universe owner of the {.pkg {pkg}} package.",
+            "!" = "No GitHub URL found in DESCRIPTION or the Git remotes.",
+            "i" = "Update the project configuration or provide an explicit {.arg repo_spec}."
+          ),
+          call = this_env
+        )
+      }
+    )
+    repo_spec <- parse_repo_url(github_url)[["repo_spec"]]
+  }
+
+  owner <- parse_repo_spec(repo_spec)[["owner"]]
+  src <- glue("https://{owner}.r-universe.dev/{pkg}/badges/version")
```

Comment:
I can't tell if the best form is what I currently have:

```
https://{owner}.r-universe.dev/{pkg}/badges/version
```

Or:

```
https://{owner}.r-universe.dev/badges/{pkg}
```

It feels like the prose and the example here sort of contradict each other?

https://docs.r-universe.dev/publish/set-up.html#badge-showing-the-number-of-the-deployed-version

Reply:
Also the markdown you get from the "copy markdown" button on a page like this:

<https://r-lib.r-universe.dev/badges>

looks like:

```
[![usethis status badge](https://r-lib.r-universe.dev/usethis/badges/version)](https://r-lib.r-universe.dev/usethis)
```